persistqueue/tests/test_sqlqueue.py

```diff
@@ -373,3 +373,11 @@ def consumer(index):
                                 "not 0 for counter's index %s" % x)
 
         self.assertEqual(len(set(counter)), len(counter))
+
+    def test_unique_dictionary_serialization(self):
```

Comment:
The code change looks great, can you please add 2 tests for json and msgpack, as the default serialier is pickle:

```python
    def __init__(self, path, name='default', multithreading=False,
                 timeout=10.0, auto_commit=True,
                 serializer=persistqueue.serializers.pickle):
```

Reply:
Added.